diablo/models/room.py

```diff
@@ -107,20 +119,18 @@ def get_room_capability_options(cls):
         }
 
     def to_api_json(self):
-        all_options = self.get_room_capability_options()
-        if self.capability:
-            capability_name = all_options[self.capability]
-            capture_options = all_options
-            if self.capability == 'screencast':
-                capture_options.pop('screencast_and_video')
-        else:
-            capability_name = None
-            capture_options = None
+        recording_type_options = {
+            'presentation_audio': 'Presentation + Audio',
+        }
+        if self.is_auditorium:
+            recording_type_options['presenter_audio'] = 'Presenter + Audio'
+            recording_type_options['presenter_presentation_audio'] = 'Presenter + Presentation + Audio'
```

Comment:
'auditorium' reqts explained in https://jira.ets.berkeley.edu/jira/browse/DIABLO-20